hackathon_site/dashboard/frontend/src/components/Filter.module.scss

```diff
@@ -1,5 +1,5 @@
-@import "./../../../assets/abstracts/variables";
-@import "./../../../assets/abstracts/mixins";
+@import "./../assets/abstracts/variables";
+@import "./../assets/abstracts/mixins";
 
```

Comment:
It would be better to create a separate folder called like "global_css" under components. And we put all the CSS files that might be used for multiple components there. How do you think Karandeep and @YuYing-Liang